chain/repository.go

```diff
@@ -410,7 +410,7 @@ func (r *Repository) GetBlockReceipts(id thor.Bytes32) (tx.Receipts, error) {
 
 // IsNotFound returns if the given error means not found.
 func (r *Repository) IsNotFound(err error) bool {
-	return err == errNotFound || r.db.IsNotFound(err)
+	return err == ErrNotFound || r.db.IsNotFound(err)
```

Comment:
Exported error here, and for this one it is exported, you can do `repo.IsNotFound(err)` to check if it's the not found error.

Reply:
I've changed it back to private, didn't realize the remaining usages were in the same package. Thanks